src/pages/Tx/TaxRateAmount.tsx

```diff
@@ -0,0 +1,31 @@
+import { useIsClassic } from "../../contexts/ChainsContext";
+import format from "../../scripts/format";
+import { getTaxData } from "../../scripts/utility";
+import { get } from "lodash";
+
+interface Props {
+  logs: Log[] | undefined
+}
+
+const TaxRateAmount = ({ logs }: Props) => {
+  const isClassic = useIsClassic();
+
+  if(!isClassic){
+    return <></>;
+  }
+
+  const tax = get(logs, "[0].log.tax") || get(logs, "[1].log.tax") || "",
+    { denom, amount } = getTaxData(tax) || {};
+
+  if (!denom || !amount) {
+    return <>-</>;
+  }
+
+  return (
+    <>
+      {format.amount(amount)} {format.denom(denom, isClassic)}
```

Comment:
Please use this component 
`<Amount denom={denom}>{amount}</Amount>`

Reply:
Done.